uqcsbot/utils/snailrace_utils.py

```diff
@@ -0,0 +1,146 @@
+import discord, random, datetime, asyncio
+
+# Racer Icon
+SNAILRACE_SNAIL_EMOJI = "🐌"
+
+# Entry opening time
+SNAILRACE_OPEN_TIME = 10
+SNAILRACE_STEP_TIME = 1
+
+# The number of characters in the track
+SNAILRACE_TRACK_LENGTH_CHARS = 15
+
+# The length of the track in steps
+SNAILRACE_TRACK_LENGTH = 100
+
+# The maximum and minimum number of steps a snail can take in a single turn
+SNAILRACE_MAX_STEP = 15
+SNAILRACE_MIN_STEP = 2
+
+# snailrace channels
+BLESSED_CHANNELS = ["bot-testing"]
+
+# Messages
+SNAILRACE_CHANNEL_ERR = "Sorry, but this command can only be used in the following channels: %s" % ", ".join(map(lambda x: "#%s" % x, BLESSED_CHANNELS))
+SNAILRACE_ENTRY_ERR = "There is currently a race going on! Please wait until the current race finishes."
+SNAILRACE_ENTRY_MSG = "The race is currently open for entry! Entry is open for the next %s seconds" % SNAILRACE_OPEN_TIME
+SNAILRACE_ENTRY_CLOSE = "Entry is now closed. Let's Race!"
+SNAILRACE_JOIN = "%s has Joined the Race!"
+SNAILRACE_ALREADY_JOINED = "%s is already apart of the race."
+SNAILRACE_BOARD = "And they're off...\n```\n%s\n```"
```

Comment:
Change requested (but feel free to dispute):

There's a disappointing lack of 🏁 on this racetrack...

Reply:
I didnt add it on the race track, but I thought it would be a good idea to add 🏁 to the end of their name when the snail has crossed the line.

Reply:
lgtm